aks/variables.tf

```diff
@@ -25,7 +25,7 @@ variable "cluster_name" {
 }
 
 variable "kubernetes_version" {
-  default     = "1.27.3"
+  default     = "1.27"
```

Comment:
Please update the AKS readme to reflect this change

Reply:
Fixed